canonical-web-and-design/ubuntu.com
# Add jest with example tests

## Done

Add Jest to project for JavaScript testing

## QA

- Check out this feature branch
- Run `./run exec yarn test-js`
- Check that two tests are run with one passing and one failing

## Issue / Card

Fixes #6943
